src/apis/cart.js

```diff
@@ -0,0 +1,22 @@
+import { instance } from './index'
+
+export const addCart = (items) => {
+    return instance.post("./carts/add", items)
```

Comment:
`.`이 잘못 들어가 있는 것 같습니다!

```suggestion
    return instance.post("/carts/add", items)
```